Formula/tbb.rb

```diff
@@ -16,10 +16,6 @@ class Tbb < Formula
   depends_on "swig" => :build
   depends_on "python@3.9"
 
-  # Fix installation of Python components
-  # See https://github.com/oneapi-src/oneTBB/issues/343
-  patch :DATA
```

Comment:
The commit the patch is based on isn't in this release yet: https://github.com/oneapi-src/oneTBB/commit/9e15720bc7744f85dff611d34d65e9099e077da4

We'll need to backport it again.